Sources/Testing/Parameterization/TypeInfo.swift

```diff
@@ -265,9 +265,11 @@ extension TypeInfo {
     }
     switch _kind {
     case let .type(type):
-      // _mangledTypeName() works with move-only types, but its signature has
-      // not been updated yet. SEE: rdar://134278607
+#if compiler(>=6.1)
+      return _mangledTypeName(type)
```

Comment:
I see the `#if compiler()` guard, but does this not have differing API availability?

Reply:
Nope, see https://github.com/swiftlang/swift/pull/76041